envs/pytorch-env.yaml

```diff
@@ -1,12 +1,12 @@
-{% if 'cuda' in build_type %}
+{% if build_type == 'cuda' %}
 channels:
   - https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
 {% endif %}
 
 packages:
   - feedstock : pytorch
   - feedstock : numactl
-{% if 'cuda' in build_type %}
+{% if build_type == 'cuda' %}
```

Comment:
Hmm..while we are changing this, should we make the PyTorch and TF env files similar in how they mark CUDA only packages?
This file uses `if` statements while the TF ones marks each line indiividually.

Reply:
OK, I changed it to using the if statements if there are more than 1 packages that need the conditional.